### ISSUE TITLE
about using dataset for train/val

### ISSUE THREAD

**Issue by ArchWing** [chiphuyen/stanford-tensorflow-tutorials#106]
I have a question,
we use dataset api to run  code in train and val preoces,
in every epoch we run the corresponding initializer
but in this way, every epoch the training set is the same(if I set shuffle seed to a constant)
but I want to shuffle them in every epoch, what should I do?(I set shuffle seed)
